cmd/go-quai/main.go

```diff
@@ -236,7 +235,7 @@ func prepare(ctx *cli.Context) {
 		netname = "Colosseum testnet"
 	}
 
-	welcome := fmt.Sprintf("Starting Quai %s on %s", params.Version.Full(), netname)
+	welcome := fmt.Sprintf("Starting Quai %s on %s", ctx.App.Version, netname)
```

Comment:
Good idea! Are any of the `params.Version...` utils or even valid anymore? If not, we should remove them, but that can be done another time.